src/Bridges/bridgeoptimizer.jl

```diff
@@ -227,18 +227,42 @@ function MOI.get(b::AbstractBridgeOptimizer,
                  attr::MOI.AbstractConstraintAttribute,
                  ci::CI)
     if is_bridged(b, typeof(ci))
-        if MOI.is_set_by_optimize(attr)
-            MOI.get(b, attr, bridge(b, ci))
-        else
-            MOI.get(b.bridged, attr, ci)
-        end
+        return MOI.get(b, attr, bridge(b, ci))
+    else
+        return MOI.get(b.model, attr, ci)
+    end
+end
+function MOI.supports(b::AbstractBridgeOptimizer,
+                      attr::MOI.AbstractConstraintAttribute,
+                      IndexType::Type{CI{F, S}}) where {F,S}
+    if is_bridged(b, IndexType)
+        return MOI.supports(b, attr, concrete_bridge_type(b, F, S))
+    else
+        return MOI.supports(b.model, attr, IndexType)
+    end
+end
+
+function MOI.set(b::AbstractBridgeOptimizer,
+                 attr::MOI.AbstractConstraintAttribute,
+                 index::CI, value)
+    if is_bridged(b, typeof(index))
+        return MOI.set(b, attr, bridge(b, index), value)
     else
-        MOI.get(b.model, attr, ci)
+        return MOI.set(b.model, attr, index, value)
     end
 end
-## Setting names
+## Getting and Setting names
+function MOI.get(b::AbstractBridgeOptimizer, attr::MOI.ConstraintName,
+                 constraint_index::CI)
+    if is_bridged(b, typeof(constraint_index))
+        return MOI.get(b.bridged, attr, constraint_index)
+    else
+        return MOI.get(b.model, attr, constraint_index)
+    end
+end
+
 function MOI.supports(b::AbstractBridgeOptimizer, attr::MOI.ConstraintName,
-                      Index::Type{<:CI})
+                      Index::Type{CI{F, S}}) where {F,S}
```

Comment:
You can leave `<:CI`

Reply:
Julia throws an "ambiguous name" method error if I leave `<:CI`.